modules/utils/2.0/utils.smk

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Snakefile Author:    Helena Winata, Bruno Grande
+# Module Author:                Bruno Grande
+# Additional Contributors:      N/A
+
+
+##### SETUP #####
+
+import sys, os
+from os.path import join
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+_UTILS = config["lcr-modules"]["utils"]
+LOG = "/logs/" + op._session.launched_fmt
+
+wildcard_constraints:
+    prefix = "[0-9]{2}-.*"
+
+
+##### RULES #####
+# _utils_bam_sort: Sort a BAM file using coordinates
+rule _utils_bam_sort:
+    input:
+        bam = "{out_dir}/{prefix}/{suffix}.bam"
+    output:
+        bam = "{out_dir}/{prefix}/{suffix}.sort.bam"
+    log:
+        stdout = "{out_dir}" + LOG + "/{prefix}/{suffix}_sort.stdout.log",
+        stderr = "{out_dir}" + LOG + "/{prefix}/{suffix}_sort.stderr.log"
+    wildcard_constraints:
+        prefix = ".*(sort).*"
```

Comment:
Can you explain why you added this? Does this override the global constraint you specified for the `prefix` wildcard?